kernel/os/include/os/util.h

```diff
@@ -26,4 +26,8 @@
 #define POINTER_TO_INT(p) ((int) ((intptr_t) (p)))
 #define INT_TO_POINTER(u) ((void *) ((intptr_t) (u)))
 
+/* Helper to retrieve pointer to "parent" object in structure */
+#define CONTAINER_OF(ptr, type, field) \
```

Comment:
For this particular macro, I'd prefer container_of in small caps. To match offsetof(). But I'm not hung up on it.

Reply:
I also considered it as a lowercase for the same reason and also this is how it's defined in Linux kernel - but since it's still not full consistent with `offsetof` (i.e. without underscore) I eventually did this as in Zephyr which has the advantage that we port some code from there so there will be less changes required :-)